.github/workflows/performance-regression.yml

```diff
@@ -97,21 +100,24 @@ jobs:
 
       - name: Running performance benchmark
         working-directory: ./ion-python-master
+        env:
+          warmups: 1000
```

Comment:
How long does 1000 warmups take? If it's longer than 30 seconds or so, that's probably enough time for us to achieve as much consistency as we're going to get. Can we shift some of these warmups to timed iterations?